Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.py

```diff
@@ -95,6 +95,61 @@ def get_user(self, user_id):
         return response
 
 
+def fetch_incidents(client: MsGraphClient, fetch_time: str, fetch_limit: int, filter_param: str, providers_param: str) \
```

Comment:
we should consider calling fetch incidents in test module(without calling last run) when fetch incidents is marked. this wy you will see the errors in the test if there are problems with the fetch configs.

Reply:
Done. When trying to fetch incidents with an invalid filter format an error is returned as we wanted. Currently I just show the error returned by MS Graph (as was done in other parts in the integration as well), i.e. an error 400 is displayed. Do you want me to handle the error and display a different message or is it informative enough?
![image](https://user-images.githubusercontent.com/61732335/82232751-b0ff6500-9937-11ea-88f7-a7091ec6f35a.png)